docs/learning/workshop_a_precision_converter_fpga_integration_journey/index.rst

```diff
@@ -0,0 +1,693 @@
+A Precision Converter FPGA Integration Journey
+==============================================
+
+Agenda
+~~~~~~
+
+1. Introduction
+2. SPI Engine Architecture
+3. What Is the Use Case?
+4. System Build
+5. System Evaluation
+6. Conclusions
+
+Introduction
+~~~~~~~~~~~~
+
+**Customer Journey**
+
+.. figure:: intro_customer_journey_1.png
+   :align: center
+
+Tools / Platforms are a customer choice, ADI ports to “current” development
+environment/kit.
+
+.. figure:: intro_customer_journey_2.png
+   :align: center
+
+Maintenance is an ADI burden. Customer don’t start their designs at the same
+time, and want to use the latest and greatest.
+
+**COS Reference Design “Donut Hole” Strategy**
+
+*Surround others processor, FPGA, micro ADI components*
+*Reference design creates stickiness*
+*Customer experiences low friction*
+*Leverage ecosystems*
+*Develop/participate in viable communities / ecosystems:*
+- Linux kernel (1.3 Billion users)
+- GitHub (40 Million users)
+- Python (100 Million users)
+- MATLAB (1 Million users)
+
+.. figure:: intro_donut.png
+   :align: center
+
+**COS Full Stack High Level Overview**
+
+.. figure:: intro_full_stack_overview.png
+   :align: center
+
+**COS Full Stack HDL Designs**
+
+*Common HDL across all IIO reference designs*
+
+*Full stack reference Designs include JESD204 physical layers (XCVR), link layers, and transport layers.*
+- Termination to DMA via an AXI-STREAM or FIFO interface
+- Runs across different Intel and Xilinx carriers
```

Comment:
```suggestion
*Full stack reference Designs include JESD204 physical layers (XCVR), link layers, and transport layers.*

- Termination to DMA via an AXI-STREAM or FIFO interface
- Runs across different Intel and Xilinx carriers
```
Without that blank line, they're not rendered properly. This should be done everywhere where you want a list